CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ a nice looking [Changelog](http://keepachangelog.com).
 
 ## Version [Unreleased]
 
+## Version [1.3.2] <small>2019-07-13</small>
+
+* fix includes to include all associations
+* fix bin/console to now actually run
+* select no longer munges field attribute
+* support virtual attributes in left_outer_joins 
```

Comment:
@kbrock are these new functionality or bug fixes?  It sounds like the last might be a never before supported feature, therefore, I think it should be a minor release, 1.4.0, right?

Reply:
@jrafanie includes, bin/console, and select are definitely bug fixes

supporting [left_outer_join](https://github.com/ManageIQ/activerecord-virtual_attributes/pull/32#) is questionable